src/Abstractions/NexusMods.Abstractions.Library/ILibraryService.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.ObjectModel;
+using DynamicData.Kernel;
+using JetBrains.Annotations;
+using NexusMods.Abstractions.Downloads;
+using NexusMods.Paths;
+
+namespace NexusMods.Abstractions.Library;
+
+/// <summary>
+/// Represents the library.
+/// </summary>
+[PublicAPI]
+public interface ILibraryService
+{
+    /// <summary>
+    /// Gets an observable collection containing all download activities.
+    /// </summary>
+    ReadOnlyObservableCollection<IDownloadActivity> DownloadActivities { get; }
```

Comment:
Does this contain only the `Downloading` and `Paused` Activities? Or can this also contain `Cancelled` and `Completed` activities?

Currently the `DownloadService` exposes a similar thing but it contains all the downloads, including Completed and cancelled ones.

Comment should clarify what this actually contains.

Reply:
Still trying to figure out what we'd want to use this for. It'll be clearer once we start using this.

Reply:
I would guess we would use this in the UI of the InProgress page and later on in the LibraryPage as well, to show download progress on either new LibraryItems or on existing items for which we are redownloading missing files.